### DISCUSSION TITLE
Quick start on Maui not working

### DISCUSSION THREAD

**Discussion by WoistdasNiveau** [Mapsui/Mapsui#2308] [Q&A]
Dear Community!

I am confused since the sample code is also very different. I copied the quick start code for the map but in my android app i only have a white screen and thats it.

Code:
```csharp
public partial class MainPage : ContentPage
{
    public MainPage()
    {
        InitializeComponent();
        
        var mapControl = new Mapsui.UI.Maui.MapControl();
        mapControl.Map?.Layers.Add(Mapsui.Tiling.OpenStreetMap.CreateTileLayer());
        Content = mapControl;
    }
}
```

and xaml:

```
<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
             xmlns:maui="clr-namespace:Mapsui.UI.Maui;assembly=Mapsui.UI.Maui"
             x:Class="MapsUiTries.MainPage">


</ContentPage>
```

and i MauiProgram:
```csharp
public static class MauiProgram
{
    public static MauiApp CreateMauiApp()
    {
        var builder = MauiApp.CreateBuilder();
        builder
            .UseMauiApp<App>()
            .UseSkiaSharp(true)
            .ConfigureFonts(fonts =>
            {
                fonts.AddFont("OpenSans-Regular.ttf", "OpenSansRegular");
                fonts.AddFont("OpenSans-Semibold.ttf", "OpenSansSemibold");
            });

#if DEBUG
        builder.Logging.AddDebug();
#endif

        return builder.Build();
    }
}
```

Edit: How can i force display the code? I does not display the xaml code at all and the MauiProgram code not properly.

**Comment by pauldendulk** [comment]
> Edit: How can i force display the code? I does not display the xaml code at all and the MauiProgram code not properly.

I updated it for you. Use three back ticks before and after every block. You can add the 'csharp' tag after the starting tick for color highlighting.

**Comment by pauldendulk** [comment]
I don't understand your question. 

>  code is also very different.

What is different from what?

You have a white screen on Android. Does it behave properly on desktop?

Did you clone the repository or create a new MAUI app and follow the quickstart guide?
